### PR TITLE
styling changes according to figma wireframe

### PR DESCRIPTION
**Pull request by NexusX45**
## What
Changed the SiteNavbar to fit the Figma wireframe.

## Link to Issue
https://github.com/speedrun-website/speedrun.website/issues/6

### Images
Default
image

Mobile
image

### CODE REVIEW

File: components/SiteNavbar.vue [hunk 1]
```diff
@@ -1,38 +1,231 @@
...
+        mx-auto
+        container
+        flex flex-wrap flex-row-reverse
+        justify-between
+        px-5
+        py-3
+        lg:flex-row
+      "
+    >
+      <div class="flex lg:hidden">
```
Comment (maxcaplan) [inline components/SiteNavbar.vue:14]: To space the buttons out I would suggest adding `flex-row space-x-2` to the parent flex div and removing `m-1` from the child buttons. Reduces the number of classes and makes the spacing more consistent.

File: components/SiteNavbar.vue [hunk 2]
```diff
@@ -1,38 +1,231 @@
...
+            items-center
+            border border-gray-300
+            rounded
+            px-4
+            py-1
+            text-gray-600
+            hover:text-black hover:border-black hover:bg-gray-100
+          "
+          @click="mobileNavIsActive = !mobileNavIsActive"
+        >
```
Comment (maxcaplan) [inline components/SiteNavbar.vue:54]: could the padding be changed from `px-4 py-1` to `px-3 py-2` to make this button more consistent with the others?

File: components/SiteNavbar.vue [hunk 3]
```diff
@@ -1,38 +1,231 @@
...
-      <h1 class="w-full py-2 text-lg text-center font-semibold leading-6 tracking-tight lg:text-left">speedrun.website</h1>
+      </div>
+      <div
+        class="
+          flex flex-col
+          w-full
+          my-2
+          text-center
+          font-semibold
+          lg:flex-row lg:w-auto lg:my-0
```
Comment (maxcaplan) [inline components/SiteNavbar.vue:103]: For this I would also suggest using `space-y-2` on the parent flex div as apposed to applying margin to the child buttons.

File: components/SiteNavbar.vue [hunk 4]
```diff
@@ -1,38 +1,231 @@
...
+              d="M21 21l-6-6m2-5a7 7 0 11-14 0 7 7 0 0114 0z"
+            />
+          </svg>
+        </button>
+      </div>
+      <div
+        class="
+          flex flex-col
+          w-full
+          my-2
```
Comment (maxcaplan) [inline components/SiteNavbar.vue:191]: Could the `my-2` be removed? It creates a lot of white space in the hamburger menu in the mobile layout

**Review by maxcaplan — CHANGES_REQUESTED** [review]
Looking good! I just added a couple notes for tweaks, there are a couple similar ones in there about using `space-x-` and `space-y-` on parent elements as apposed to applying margin to the child elements. 
You can read more about tailwinds space between class [here](https://tailwindcss.com/docs/space), it's pretty handy.

Once those get worked out we should be good to merge!

File: components/SiteNavbar.vue (hunk 1 above)
[the remark this answers is not in the document]
Comment (NexusX45) [reply components/SiteNavbar.vue:14]: Sure!

**Comment by NexusX45** [comment]
I've made the changes, can you take a look again?

**Review by maxcaplan — APPROVED** [review]

**Comment by maxcaplan** [comment]
Things are looking good! I can go ahead and merge this unless there's something that others want added

**Comment by NexusX45** [comment]
Great!

**Comment by CapitaineToinon** [comment]
Looks good, thanks for the PR 👍

**Comment by NexusX45** [comment]
Happy to help, thanks for accepting the PR

**Pull request merged**